test/built-ins/Promise/prototype/finally/rejected-observable-then-calls-argument.js

```diff
@@ -57,6 +53,12 @@ Promise.prototype.then = function(resolve, reject) {
   );
   if (calls === 0) {
     assert.throws(MyError, resolve, '`resolve()` throws `MyError`');
+    assert.sameValue(arguments.length, 1, '`then` invoked with one argument');
+  } else {
+    assert.sameValue(isConstructor(reject), false, 'isConstructor(reject) must return false');
```

Comment:
Should we also check the `arguments.length` in the `else` branch?

Reply:
That seems slightly out-of-scope, but it wouldn't hurt.